src/main/java/kafdrop/service/KafkaMonitorImpl.java

```diff
@@ -280,8 +280,8 @@ public SearchResultsVO searchMessages(String topic,
         case EXCEEDED_MAX_SCAN_COUNT:
           results.setCompletionDetails(
             String.format(
-              "Search timed out after scanning %d messages.  Last scanned message timestamp was %d.  Adjust your time" +
-                " span for more results.",
+              "Search timed out after scanning %d messages. Last scanned message timestamp was %2$tF %2$tT. Adjust your time" +
+              " span for more results.",
```

Comment:
```suggestion
                " span for more results.",
```

Reply:
Ha! had the opposite feeling that there are too many of them. (:

Reply:
The first parameter of `String.format` is broken in two lines. The second half is indented a bit more to show it's not the second parameter but second half of the first one.